client/core/trade_simnet_test.go

```diff
@@ -172,9 +172,15 @@ func startClients(ctx context.Context) error {
 			if err != nil {
 				return err
 			}
-			c.log("Connected %s wallet", unbip(assetID))
+			c.log("Connected %s wallet (spv = %v)", unbip(assetID), wallet.spv)
```

Comment:
Want to just rename `wallet.spv` to `wallet.created`? Upcoming dcr spv wallets via json-rpc won't have this field set to true.

Reply:
You have this in #788, so passing this to you.

Reply:
Oh, regarding the synced loop below, I think that applies to `created` really instead of `spv`, but it won't hurt for spv either.

Also, the sync status fix in https://github.com/decred/dcrdex/pull/1271 may also solve some issue and possibly eliminate the need for the sleep about  20 lines down from here.  We'll see when that goes in I guess.